lti_consumer/lti_1p3/consumer.py

```diff
@@ -161,6 +163,47 @@ def set_launch_presentation_claim(
             },
         }
 
+    def set_context_claim(
+            self,
+            context_id,
+            context_types=None,
+            context_title=None,
+            context_label=None
+    ):
+        """
+        Optional: Set context claims
+
+        https://www.imsglobal.org/spec/lti/v1p3/#context-claim
+
+        Arguments:
+            context_id (string):  Unique value identifying the user
+            context_types (list):  A list of context type values for the claim
+            context_title (string):  Plain text title of the context
+            context_label (string):  Plain text label for the context
+        """
+        if not isinstance(context_types, list):
```

Comment:
I think we should go for consistent inputs instead of accepting both a list and a string here.

Reply:
Done in next commit. Raising a `TypeError` if a list is not passed in